packages/webb-ui-components/src/components/Avatar/Avatar.tsx

```diff
@@ -32,6 +32,7 @@ export const Avatar: React.FC<AvatarProps> = (props) => {
     darkMode,
     fallback,
     size = 'md',
+    sourceVariant = 'uri',
```

Comment:
Sorry! But do you think we need to change the default `sourceVariant` is `address`? Because right now, most of our implementation is using address. @AhmedKorim

Reply:
it's okay to change them the occurrence did it on other PRs